pkg/cidata/cidata.TEMPLATE.d/boot/04-persistent-data-volume.sh

```diff
@@ -16,11 +16,14 @@ if [ "$(awk '$2 == "/" {print $3}' /proc/mounts)" == "tmpfs" ]; then
 	if [ -e /dev/disk/by-label/data-volume ]; then
 		# Find which disk is data volume on
 		DATA_DISK=$(blkid | grep "data-volume" | awk '{split($0,s,":"); sub(/\d$/, "", s[1]); print s[1]};')
-		# Automatically expand the data volume filesystem
-		growpart "$DATA_DISK" 1 || true
-		# Only resize when filesystem is in a healthy state
-		if e2fsck -f -p /dev/disk/by-label/data-volume; then
-			resize2fs /dev/disk/by-label/data-volume
+		# growpart command may be missing in older VMs
+		if command -v growpart >/dev/null 2>&1; then
+			# Automatically expand the data volume filesystem
+			growpart "$DATA_DISK" 1 || true
+			# Only resize when filesystem is in a healthy state
+			if e2fsck -f -p /dev/disk/by-label/data-volume; then
+				resize2fs /dev/disk/by-label/data-volume
```

Comment:
Even if resize2fs failed, the following `mount` has to be executed

Reply:
Sure, added `|| true`